setup.py

```diff
@@ -298,10 +298,12 @@ class develop(setuptools_develop.develop):
     def run(self, *args, **kwargs):
         _check_rust()
         build = self.get_finalized_command('build')
+        build_base = pathlib.Path('build').resolve()
```

Comment:
Use `build.build_base` here.

Reply:
Rather than fixing this. I've updated all `Path('build')` invocations to use `build_base` everywhere as a separate commit.